drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInViewModelFactory.kt

```diff
@@ -63,4 +68,17 @@ internal class DropInViewModelFactory(
         @Suppress("UNCHECKED_CAST")
         return DropInViewModel(bundleHandler, orderStatusRepository, analyticsRepository) as T
     }
+
+    internal fun PaymentMethodsApiResponse.overridePaymentMethodInformation(
```

Comment:
This method probably doesn't need to be `internal` if it's inside `DropInViewModelFactory`

Reply:
This will also be fixed with the next commit. I moved it out from the `DropInViewModelFactory`.